app/dts/bindings/behaviors/zmk,behavior-key-toggle.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 The ZMK Contributors
```

Comment:
```suggestion
# Copyright (c) 2022 The ZMK Contributors
```

Perhaps update copyright since we're ideally merging this within the year?

Reply:
Done, thanks!